create-mern-ts-app.js

```diff
@@ -43,6 +44,9 @@ function createProjectTemplate(projectName) {
   console.log(chalk.cyan(destRoot + '\n'));
   fs.mkdirsSync(destRoot);
   fs.copySync(srcRoot, destRoot);
+  let envContents = fs.readFileSync(path.join(destRoot, 'backend', '.env.example'), 'utf8')
```

Comment:
Would you mind doing two extra things here please, shouldn't take much work:

First would be to replace the comments on top of the generated `.env` file with something like
```
# This is an auto-generated file, change at your own will and risk.
```

Second, would you mind separating this into a new function, for example, `generateEnvFile()` to be called right after `createProjectTemplate` in the end of the script?